sklearn/decomposition/online_lda.py

```diff
@@ -562,6 +564,7 @@ def transform(self, X):
 
         doc_topic_distr, _ = self._e_step(X, cal_sstats=False,
                                           random_init=False)
+        doc_topic_distr = normalize(doc_topic_distr, axis=1, norm='l1')
```

Comment:
This looks correct, but it's simpler to just divide by the sum (since the inputs are non-negative anyway):

```py
doc_topic_distr /= doc_topic_distr.sum(axis=1)
```